lib/bugsnag/resque.rb

```diff
@@ -37,7 +37,7 @@ def save
 # Auto-load the failure backend
 Bugsnag::Resque.add_failure_backend
 
-Resque.before_first_fork do
+Resque.after_fork do
   Bugsnag.configuration.app_type = "resque"
   Bugsnag.configuration.delivery_method = :synchronous
```

Comment:
I think we should avoid magic here (automatically detecting Resque and setting the config option) and let Rescue/Sidekiq users to add declare configuration setting themselves. This way it will be explicit and it won't hiddenly break someone's app by modifying global state from a `fork` hook.